doc/user_guide/examples.rst

```diff
@@ -397,14 +398,33 @@ code that is output is the same as the original (but looks different
 as it has been translated to PSyIR and then output by the PSyIR
 fortran back-end).
 
-Example 16: Kernel Extraction
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The subdirectory ``full_example_extract`` contains a runnable example
-of LFRic code that creates a NetCDF file with the input and output
-parameters of a simple kernel. This example requires the installation
-of a Fortran compiler and NetCDF development environment. After compilation
-of the code, you can run the example. which will produce a NetCDF file:
+Example 17: Runnable Simplified LFRic Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This directory contains three simplified LFRic examples, that can be
+compiled and executed - a suitable Fortran compiler is of course
+required. The examples are using a subset of the LFRic infrastructure
+library, which is contained in PSyclone and which has been slightly
+modified to make it easier to create stand-alone, non-MPI LFRic codes.
+
+Example 17.1: Very Simple Runnable Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The subdirectory ``full_example`` contains a very simple example code
+that uses PSyclone to create two single kernels. It uses unit-testing
+code from various classes to create the required data structures like
+initial grid etc.
```

Comment:
While we don't document this in the other examples (e.g. gocean), I've added this ... and fixed the mistake that it would create two nc files (we don't support builtins yet, so the first invoke statement cannot be instrumented).